### PR TITLE
Kctlread

### PR DESCRIPTION
**Pull request by SchwarzM**
Implements read commands into kubernikusctl to interact without a gui

### CODE REVIEW

**Comment by BugRoger** [comment]
rebase and quash please

**Comment by SchwarzM** [comment]
done

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 bin/
 _scratch
 _output
+*.swp
```
Comment (BugRoger) [inline .gitignore]: https://github.com/BugRoger/dotfiles/blob/master/vimrc#L47-L49 😅

**Comment by BugRoger** [comment]
What was the problem with the `k.client == nil` issue the other day?

**Comment by SchwarzM** [comment]
TBH i don't really know i moved the init into the PreRun of the cluster command and it worked.
I would still like it to be in PersistentPreRun of get cause all get commands will need it.

**Pull request merged**